### PR TITLE
Add Oblivion Support

### PR DESCRIPTION
**Pull request by SirStig**
Added Oblivion Support:

- Created a oblivion-post-deploy.sh script

- Created a oblivion-post-deploy.desktop script

- Modified install-vortex.sh script to create a Symlink of oblivion-post-deploy.desktop to steam deck desktop.

- Modified post-deploy.sh to recreate or create all .desktop symlinks

### CODE REVIEW

**Comment by pikdum** [comment]
Very cool.

Might be neat to make these desktop shortcuts in `post-install.sh` too, if Vortex is installed and the games are too.

**Comment by SirStig** [comment]
> Very cool. Will test and merge later today.
> 
> 
> 
> Might be neat to make these desktop shortcuts in `post-install.sh` too, if Vortex is installed and the games are too.
> 
> That way people would get the new shortcuts just by running the update shortcut.


Ah didn’t check all the .sh

File: post-install.sh [hunk 1]
```diff
@@ -8,4 +8,10 @@ if [ ! -f "$HOME/.local/share/applications/vortex.desktop" ]; then
     ln -s ~/.pikdum/steam-deck-master/vortex/install-vortex.desktop ~/Desktop/install-vortex.desktop || true
 fi
 
+ln -sf ~/.local/share/applications/vortex.desktop ~/Desktop/
+ln -sf ~/.pikdum/steam-deck-master/vortex/skyrim-post-deploy.desktop ~/Desktop/
+ln -sf ~/.pikdum/steam-deck-master/vortex/fallout4-post-deploy.desktop ~/Desktop/
+ln -sf ~/.pikdum/steam-deck-master/vortex/falloutnv-post-deploy.desktop ~/Desktop/
+ln -sf ~/.pikdum/steam-deck-master/vortex/oblivion-post-deploy.desktop ~/Desktop/
```
Comment (pikdum) [inline post-install.sh:15]: **suggestion:** Maybe this should all be wrapped in a check that `~/.local/share/applications/vortex.desktop` exists, so they don't show up when Vortex isn't installed yet.

Comment (SirStig) [reply post-install.sh:15]: I can do that. Sorry was a simple edit from my phone.

**Review by pikdum — APPROVED** [review]

**Pull request merged**